sanic/request.py

```diff
@@ -426,28 +426,40 @@ def credentials(self) -> Optional[Credentials]:
                 pass
         return self.parsed_credentials
 
+    def get_form(
+        self, keep_blank_values: bool = False
+    ) -> Optional[RequestParameters]:
+        self.parsed_form = RequestParameters()
+        self.parsed_files = RequestParameters()
+        content_type = self.headers.getone(
+            "content-type", DEFAULT_HTTP_CONTENT_TYPE
+        )
+        content_type, parameters = parse_content_header(content_type)
+        try:
+            if content_type == "application/x-www-form-urlencoded":
+                self.parsed_form = RequestParameters(
+                    parse_qs(
+                        self.body.decode("utf-8"),
+                        keep_blank_values=keep_blank_values,
+                    )
+                )
```

Comment:
Should we consider caching the parsed for with the different settings like we do for query args?
https://github.com/sanic-org/sanic/blob/3a6cc7389c5166a0c9580daf5d709a0dee85a996/sanic/request.py#L561-L563

Reply:
I've pretty much burned my weekend on this, if we want to make additional changes to functionality someone can pick it up from here.

Reply:
That to me sounds like its own PR and discussion.